src/stream/src/executor/mview/materialize.rs

```diff
@@ -577,10 +593,17 @@ impl<SD: ValueRowSerde> MaterializeCache<SD> {
     ) -> StreamExecutorResult<()> {
         let mut futures = vec![];
         for key in keys {
+            self.metrics
+                .materialize_cache_total_count
+                .with_label_values(&[&self.actor_id])
+                .inc();
             if self.data.contains(key) {
+                self.metrics
+                    .materialize_cache_hit_count
+                    .with_label_values(&[&self.actor_id])
```

Comment:
Here I show this metrics on actor_id level, just like `stream_join_lookup_miss_count`. Not sure whether we need to refactor join's cache miss metrics to executor's level, because there may be serval join in an actor.